analysis/pf_dataset_definition.py

```diff
@@ -0,0 +1,39 @@
+from ehrql import INTERVAL, Measures, months, create_dataset
+from ehrql.tables.tpp import patients, clinical_events, practice_registrations
+
+from pf_dataset import (
+    get_uncomplicated_uti_denominator,
+    get_shingles_denominator,
+    get_impetigo_denominator,
+    get_infected_insect_bites_denominator,
+    get_acute_sore_throat_denominator,
+    get_acute_sinusitis_denominator,
+    get_acute_otitis_media_denominator,
+)
+
+from measures_definition_pf_codes_conditions import ( 
+    pharmacy_first_ids,
+)
+
+import codelists
+
+
+index_date = "2024-01-01"
+dataset = create_dataset()
+
+selected_events = (clinical_events.where(
+    clinical_events.date.is_on_or_before(index_date))
+.where(clinical_events.consultation_id.is_in(pharmacy_first_ids))
+)
```

Comment:
I know this is slightly different to the measures definition because we are using `is_on_or_between()` but maybe there is a way to move this also to `pf_dataset`?